app/views/spree/admin/products/_shipping_category_form.html.haml

```diff
@@ -1,4 +1,4 @@
 = f.field_container :shipping_categories do 
   = f.label :shipping_category_id, t(:shipping_category)
-  = f.collection_select(:shipping_category_id, Spree::ShippingCategory.all, :id, :name, {:include_blank => true}, {:class => 'select2 fullwidth'})
+  = f.collection_select(:shipping_category_id, Spree::ShippingCategory.all, :id, :name, {:include_blank => false, :selected => DefaultShippingCategory.find_or_create.id}, {:class => 'select2 fullwidth'})
```

Comment:
This selects "Default" no matter what, which would be a problem in this scenario:

1. In the new product form, fill in required product information except product name.
2. Select a non-default shipping category.
3. Submit the form.
4. In the product form showing the errors, the "Default" shipping category is selected even when it should be a different one.

How about setting the default category for the object in `spree/admin/products#new`? (`app/controllers/spree/admin/products_controller_decorator.rb`). Something like:

```ruby
def new
  @object.shipping_category = DefaultShippingCategory.find_or_create
  super
end
```

Then, this wouldn't be called after the user's unsuccessful submission of the form (`#create`).

Reply:
Hi @kristinalim - thanks, this definitely makes sense. I had been thinking of a front-end approach with just the forms, instead of the object itself. I see the behavior you're describing in both forms upon submission. This seems to be working now, but I'll retest everything before submitting the PR again.

Regarding the question of the shipping method spec above (adding a second shipping category), I believe this has been resolved with the changes that I made to the shipping category factory.